src/components/Category Cards/Category.jsx

```diff
@@ -0,0 +1,23 @@
+import "./Category.css";
+import { CategoriesData } from "../../data/Homedata";
+
+const CategoriesCards = () => {
+  return (
+    <>
+      {CategoriesData.map((e) => (
+        <main
+          style={{ backgroundColor: e.color }}
+          className='category-card card-shadow mr-1'>
+          <section className='category-image-container'>
+            <img src={e.image} alt='Category' className='category-image' />
+          </section>
+          <h2 className='ml-1'>Shop</h2>
+          {console.log(e)}
```

Comment:
remember to remove console logs once its no more needed and before pushing the code